Dockerfile

```diff
@@ -8,6 +8,13 @@ RUN set -eux; \
     sudo mkdir -p /usr/lib/jvm/oracle8; \
     curl -L --fail "https://javadl.oracle.com/webapps/download/AutoDL?BundleId=246284_165374ff4ea84ef0bbd821706e29b123" | sudo tar -xvzf - -C /usr/lib/jvm/oracle8 --strip-components 1
 
+# GraalVM with native image support
+FROM ghcr.io/graalvm/graalvm-ce:ol7-java11-22.2.0 AS graalvm-native-image-jdk11
```

Comment:
`ol8-java11-22.2.0` also exists if we want to keep the same version of "OracleLinux" as below